lib/executor/ramping_vus.go

```diff
@@ -107,62 +107,66 @@ func (vlvc RampingVUsConfig) Validate() []error {
 // execution steps would look something like this:
 //
 // VUs  ^
-//    10|          *
-//     9|         ***
-//     8|        *****
-//     7|       *******
-//     6|      *********
-//     5|     ***********
-//     4|    *************
-//     3|   ***************
-//     2|  *****************
-//     1| *******************
-//     0------------------------> time(s)
-//       01234567890123456789012   (t%10)
-//       00000000001111111111222   (t/10)
+//
+//	10|          *
+//	 9|         ***
+//	 8|        *****
+//	 7|       *******
+//	 6|      *********
+//	 5|     ***********
+//	 4|    *************
+//	 3|   ***************
+//	 2|  *****************
+//	 1| *******************
+//	 0------------------------> time(s)
```

Comment:
```suggestion
//	 0+-----------------------> time(s)
```
Wouldn't that be better :thinking:

Reply:
not really, it might confuse someone